src/lib/iso8601-duration/index.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Borrowed from https://github.com/tolu/ISO8601-duration/blob/master/src/index.js
```

Comment:
Would be good to document exactly what version so that people upgrading later can look up changelogs etc